### PR TITLE
fix (ATracker): add support for Cloud Logs target type

### PR DESCRIPTION
**Pull request by veenanr**
## PR summary
Add support for a new ATracker target type (cloud-logs) and its endpoint fields

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Current vs new behavior  
ATracker will support a new target type (cloud-logs) in addition to the existing target types

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

## Other information
Unit tests results:

```
veenarao$ go test
go: downloading github.com/onsi/gomega v1.31.1
Running Suite: AtrackerV2 Suite
===============================
Random Seed: 1712004023
Will run 94 of 94 specs

••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••
Ran 94 of 94 Specs in 4.006 seconds
SUCCESS! -- 94 Passed | 0 Failed | 0 Pending | 0 Skipped
PASS
ok  	github.com/IBM/platform-services-go-sdk/atrackerv2	5.064s

```

Integration test results:

```
veenarao$ go test github.com/IBM/platform-services-go-sdk/atrackerv2 -tags="integration" -v
=== RUN   TestAtrackerV2
Running Suite: AtrackerV2 Suite
===============================
Random Seed: 1712004111
Will run 137 of 137 specs

••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••
------------------------------
• [SLOW TEST:7.479 seconds]
AtrackerV2 Integration Tests
/Users/veenarao/go/src/github.com/IBM/platform-services-go-sdk/atrackerv2/atracker_v2_integration_test.go:42
  DeleteTarget - Delete a target
  /Users/veenarao/go/src/github.com/IBM/platform-services-go-sdk/atrackerv2/atracker_v2_integration_test.go:756
    DeleteTarget(deleteTargetOptions *DeleteTargetOptions)
    /Users/veenarao/go/src/github.com/IBM/platform-services-go-sdk/atrackerv2/atracker_v2_integration_test.go:804
------------------------------

Ran 137 of 137 Specs in 16.669 seconds
SUCCESS! -- 137 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestAtrackerV2 (16.71s)
PASS
ok  	github.com/IBM/platform-services-go-sdk/atrackerv2	17.733s
```

### CODE REVIEW

File: atrackerv2/atracker_v2_integration_test.go [hunk 1]
```diff
@@ -1,7 +1,8 @@
 //go:build integration
+// +build integration
 
 /**
- * (C) Copyright IBM Corp. 2023.
+ * (C) Copyright IBM Corp. 2022.
```
Comment (padamstx) [inline atrackerv2/atracker_v2_integration_test.go:5]:
```suggestion
 * (C) Copyright IBM Corp. 2024.
```

File: atrackerv2/atracker_v2_examples_test.go [hunk 2]
```diff
@@ -1,7 +1,8 @@
 //go:build examples
+// +build examples
 
 /**
- * (C) Copyright IBM Corp. 2023.
+ * (C) Copyright IBM Corp. 2022.
```
Comment (padamstx) [inline atrackerv2/atracker_v2_examples_test.go:5]:
```suggestion
 * (C) Copyright IBM Corp. 2024.
```

**Review by padamstx — CHANGES_REQUESTED** [review]
Everything looks good, except the copyright year in a couple files.  Could you fix that please?

File: atrackerv2/atracker_v2_integration_test.go (hunk 1 above)
Comment (veenanr) [reply atrackerv2/atracker_v2_integration_test.go]: Updated

File: atrackerv2/atracker_v2_examples_test.go (hunk 2 above)
Comment (veenanr) [reply atrackerv2/atracker_v2_examples_test.go]: Updated

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**